.grabl/automation.yml

```diff
@@ -98,4 +104,10 @@ release:
         cat VERSION
         export DEPLOY_APT_USERNAME=$REPO_VATICLE_USERNAME
         export DEPLOY_APT_PASSWORD=$REPO_VATICLE_PASSWORD
+        # Downgrading Python is needed to work around bazelbuild/rules_pkg#397
```

Comment:
The comment should be like this I think:

```
# Do not upgrade Python until bazelbuild/rules_pkg#397 is fixed
```

The current comment will be confusing when read without context